docs/superpowers/specs/2026-05-24-app-review-popup-design.md

```
@@ -0,0 +1,165 @@
+---
+
+## 개요
+
+스토어 별점/리뷰 유도를 위한 2단계 팝업 시스템 구현.  
+긍정적 사용자만 필터링해 스토어 리뷰로 유도하고, 부정적 사용자는 쿨타임 처리.
+
+---
+
+## 플로우
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)